Inspector/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "web-driver-inspector",
   "version": "1.0.0",
   "description": "",
```

Comment:
This is where you want to have 

`'React based web inspector to inspect current device state'`